### PR TITLE
PLU-125: [EXCEL-8] Get cell value action

### PR DESCRIPTION
**Pull request by ogp-weeloong**
## Problem
Users want to read data from their excel file.

## Solution
This implements a simple "get cell value" action that allows for getting values of up to **3 cells**.

At some point, we will need something like a "get first table row matching some criteria" action, but these advanced actions will later; it's not straightforward to implement without bashing our rate limits.

## Tests
- Check that I can read data from specified cells
- Check for read what you write consistency - put this action after a "create table row" action, and check that I can read what was written, and that formula output is also consistent.

### CODE REVIEW

File: packages/backend/src/apps/m365-excel/actions/get-cell-values/index.ts [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+      label: 'Values',
+      key: 'cells',
+      type: 'multirow' as const,
+      required: true,
+      // We need to make 1 separate request for each cell, so limit to 3 as a
+      // balance between convenience and API quota usage.
+      description: 'Specify cells you want to get the values of. (Max 3)',
+      subFields: [
+        {
+          label: 'Cell Address (e.g. A1, B10)',
```
Comment (pregnantboy) [inline packages/backend/src/apps/m365-excel/actions/get-cell-values/index.ts]: nit: the e.g. made me think i could comma separate the cell addresses. i think (e.g. A1) is clear enough

File: packages/backend/src/apps/m365-excel/actions/get-cell-values/index.ts [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+        },
+      ],
+    },
+  ],
+
+  getDataOutMetadata,
+
+  async run($) {
+    const { fileId, worksheetId, cells: rawCells } = $.step.parameters
+    const cells = rawCells as Array<{ address: string }>
```
Comment (pregnantboy) [inline packages/backend/src/apps/m365-excel/actions/get-cell-values/index.ts:79]: trim

File: packages/backend/src/apps/m365-excel/actions/get-cell-values/index.ts [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+
+    for (const cell of cells) {
+      if (!/^[a-zA-Z]+[1-9][0-9]*$/.test(cell.address)) {
+        throw generateStepError(
+          `Invalid cell address ${cell.address}`,
+          'Please specify cell addresses in A1 notation.',
+          $.step.position,
+          $.app.key,
+        )
+      }
```
Comment (pregnantboy) [inline packages/backend/src/apps/m365-excel/actions/get-cell-values/index.ts:101]: trim cell addresses

**Review by pregnantboy — APPROVED** [review]
lgtm straightforward can address nits later

File: packages/backend/src/apps/m365-excel/actions/get-cell-values/index.ts (hunk 1 above)
Comment (ogp-weeloong) [reply packages/backend/src/apps/m365-excel/actions/get-cell-values/index.ts]: agreed, done

File: packages/backend/src/apps/m365-excel/actions/get-cell-values/index.ts (hunk 3 above)
Comment (ogp-weeloong) [reply packages/backend/src/apps/m365-excel/actions/get-cell-values/index.ts:101]: good call, done.

**Pull request merged**